GameReadyValidator/src/main/java/pl/przemyslawjakubowski/tajgertim/gamereadyvalidator/gamevalidation/ConfirmationReceivedEventListener.java

```diff
@@ -0,0 +1,34 @@
+package pl.przemyslawjakubowski.tajgertim.gamereadyvalidator.gamevalidation;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationListener;
+import org.springframework.stereotype.Component;
+import pl.jakubowskiprzemyslaw.tajgertim.models.confirmation.Confirmation;
+import pl.jakubowskiprzemyslaw.tajgertim.models.confirmation.FinalConfigurationConfirmation;
+import pl.jakubowskiprzemyslaw.tajgertim.queues.Queues;
+import pl.jakubowskiprzemyslaw.tajgertim.services.QueueService;
+
+@Component
+class ConfirmationReceivedEventListener implements ApplicationListener<ConfirmationReceivedEvent> {
+
+    private final QueueService queueService;
+    private final Confirmations confirmations;
+
+    @Autowired
+    ConfirmationReceivedEventListener(QueueService queueService, Confirmations confirmations) {
+        this.queueService = queueService;
+        this.confirmations = confirmations;
+    }
+
+    @Override
+    public void onApplicationEvent(ConfirmationReceivedEvent event) {
+        Confirmation confirmation = event.getConfirmation();
```

Comment:
inline

Reply:
We've done it this way according to articles like this one:
https://blogs.msdn.microsoft.com/cellfish/2009/08/19/object-calisthenics-rule-4-use-only-one-dot-per-line/

Reply:
Microsoft developer? )
In some cases, it makes sense to extract separate variable. But you are taking this advice too seriously and create variables that sometimes don't make any sense and only pollute your code

Reply:
inlined :)